pkgs/development/tools/build-managers/cmake/002-application-services.diff

```diff
@@ -1,20 +1,18 @@
-diff --git a/Source/CMakeLists.txt b/Source/CMakeLists.txt
```

Comment:
To reduce the diff, can you clone the CMake repo, checkout the 3.25.1 tag, remove the lines, and then `git diff`? (I'm sorry for being so pedantic here, please yell at me if you think I should just stop.)

Reply:
No problem. I am usually way more pedantic.

However, I do not generate patches by using the git repo, but the tarball provided by them. I will take a bit more time.

Reply:
Seriously, I will refrain from this. The patch is working as expected, and this is too much work for a "beautiful" patch.

Reply:
I'll do it then, if you give me a (verbal) OK to force push.

Reply:
Many thanks, @winterqt! I have solved it.

And merry nixmas!